pkg/tls/options.go

```diff
@@ -17,8 +17,8 @@ import (
 )
 
 // NewServerConfig provides new server TLS configuration.
-func NewServerConfig(logger log.Logger, cert, key, clientCA string) (*tls.Config, error) {
-	if key == "" && cert == "" {
+func NewServerConfig(logger log.Logger, certPath, keyPath, clientCA string) (*tls.Config, error) {
+	if keyPath == "" && certPath == "" {
```

Comment:
Any specific reasons for changing their var names?

Reply:
I have created a new var `certificate`, the existing name of the certificate path was `cert`, which i thought will cause some confusion. Hence changed it.